engine/dlib/src/dmsdk/dlib/spinlocktypes_mach.h

```diff
@@ -15,24 +15,25 @@
 #ifndef DM_SPINLOCKTYPES_MACH_H
 #define DM_SPINLOCKTYPES_MACH_H
 
-#include <libkern/OSAtomic.h>
+#include <os/lock.h>
```

Comment:
The other implementation was deprecated and produce spam when compiling.

New interface is available from iOS 10..0+ and macOS 10.12+
https://developer.apple.com/documentation/os/os_unfair_lock_t